src/Serilog/Context/LogContext.cs

```diff
@@ -94,8 +95,7 @@ public static IDisposable PushProperties(params ILogEventEnricher[] properties)
             var stack = GetOrCreateEnricherStack();
             var bookmark = new ContextStackBookmark(stack);
 
-            foreach (var prop in properties)
-                stack = stack.Push(prop);
+            stack = properties.Aggregate(stack, (current, prop) => current.Push(prop));
```

Comment:
Huh, I guess R# suggest this and I just Alt+Enter'd my way through :stuck_out_tongue_closed_eyes: I'll revert...